modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2678,6 +2678,192 @@ static void testYOLO(const std::string& weightPath, const std::vector<int>& refC
     normAssertDetections(refClassIds, refScores, refBoxes, keep_classIds, keep_confidences, keep_boxes);
 }
 
+TEST_P(Test_ONNX_layers, YOLOx)
```

Comment:
Is it possible to reuse `testYOLO` introduced from https://github.com/opencv/opencv/pull/24553?

Reply:
Possible, if we add if condition for each independent test that diverges from `testYOLO`. I actually wanted to do that, but decided to open PR this way to get your feed back